ADAL/src/ADAuthenticationError.m

```diff
@@ -236,8 +237,7 @@ + (ADAuthenticationError*)errorFromNonHttpsRedirect:(NSUUID *)correlationId
                                                  correlationId:correlationId];
 }
 
-+ (ADAuthenticationError *)keychainErrorFromOperation:(NSString *)operation
-                                               status:(OSStatus)status
++ (ADAuthenticationError *)keychainErrorFromOperation:(NSString *)operation                                               status:(OSStatus)status
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 145 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)